src/content-handlers/iiif/extensions/uv-aleph-extension/lib/aleph.css

```diff
@@ -1,6 +1,6 @@
 html.ios {
   --ion-default-font:
-    -apple-system, BlinkMacSystemFont, "Helvetica Neue", "Roboto", sans-serif;
+    -apple-system, blinkmacsystemfont, "Helvetica Neue", "Roboto", sans-serif;
```

Comment:
Not sure if it's right to lowercase blinkmacsystemfont either -- though I'm not familiar with this.